src/pages/components/header/menu/menu.scss

```diff
@@ -57,8 +57,12 @@
     }
 
     .main .header.open{
+        display: block;
+        position: absolute;
+        z-index: 500;
         max-width: 100%;
         padding: 30px;
+        margin-right: .51rem;
```

Comment:
why this is the right number?

Reply:
it's can be change to 0.5 rem.

Reply:
rem is related to the default font size, and I'm curios fo understand why this specific size has been chosen

Reply:
I'm use to work with rem ..
any way I change it to  px

Reply:
rem is fine - I'm just confused by the very specific number